src/views/Home/components/Banners/LotteryBanner.tsx

```diff
@@ -60,15 +62,13 @@ export const StyledSubheading = styled(Heading)`
 const isLotteryLive = (status: LotteryStatus) => status === LotteryStatus.OPEN
 
 const LotteryPrice: React.FC = () => {
-  const {
-    currentRound: { amountCollectedInCake, status },
-  } = useLottery()
+  const { data } = useSWR<LotteryResponse>(['currentLottery'])
   const cakePriceBusd = usePriceCakeBusd()
-  const prizeInBusd = amountCollectedInCake.times(cakePriceBusd)
+  const prizeInBusd = new BigNumber(data.amountCollectedInCake).times(cakePriceBusd)
   const prizeTotal = getBalanceNumber(prizeInBusd)
   const { t } = useTranslation()
 
-  if (isLotteryLive(status)) {
+  if (isLotteryLive(data.status)) {
```

Comment:
is it possible if data is `null` ?

Reply:
No, we are checking in here https://github.com/pancakeswap/pancake-frontend/blob/d495334dca7990af15d2b321cde0da35e4eb9532/src/views/Home/components/Banners/hooks/useIsRenderLotteryBanner.tsx we only render Lottery Banner when it is fetched. And fetchLottery returns object when there is an error too. https://github.com/pancakeswap/pancake-frontend/blob/01063288e7ca48f3ae1037e4d23b1cc765220404/src/state/lottery/helpers.ts#L76